src/main.ts

```diff
@@ -75,9 +78,12 @@ function isURL(str: string) {
   }
 }
 
-async function readJson(file: string) {
+async function readJson(file: string, token?: string) {
   if (isURL(file)) {
-    const { data } = await axios.get(file)
+    const headers = token ? {
+      Authorization: `Bearer ${token}`
```

Comment:
```suggestion
      Authorization: `token ${token}`
```
Shouldn't this be `token` instead of `Bearer`? 
I've seen that [here](https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#authentication) and in some Stack Overflow answers they use it like this...